docs/Makefile

```diff
@@ -5,12 +5,12 @@
 # from the environment for the first two.
 # Do not fail the build if there are warnings
 SPHINXOPTS    = -j8
-SPHINXBUILD  ?= sphinx-build
+SPHINXBUILD  ?= sphinx.cmd.build
```

Comment:
A little question: why we change sphinx build command to `sphinx.cmd.build`?

Reply:
It's a little more "right" way because we are enforcing using sphinx from the project environment (because of `python -m`) even if there is a system wide one. I can roll back because in CI it doesn't matter.

Reply:
It's ok to use the command and no need to roll back.